js/dist/wp-seo-metabox-302.js

```diff
@@ -105,13 +105,48 @@
 	 * Shows a informational popup if someone click the add keyword button
 	 */
 	function addKeywordPopup() {
-		var title = $( '#wpseo-add-keyword-popup' ).find( 'h3' ).html();
+		var $buyButton = $( '#wpseo-add-keyword-popup-button' ),
+			title = $buyButton.text(),
+			$tbWindow,
```

Comment:
Please use a variable that is a bit more descriptive, my guess is this is the thickboxwindow, the variable name could be a bit more clear.